tle/cogs/contests.py

```diff
@@ -490,7 +486,8 @@ async def ranklist(self, ctx, contest_id: int, *args: str):
                                                                                show_unofficial=show_official is False)
 
         # for Edu rounds the cf api returns the unofficial participants as official as well.
-        # Hence we need to filter in actual rated contestants
+        # Hence we need to fix ranks for actual rated people
+        # Note that unofficial people still have their original ranks but they will be filtered out in show_ranklist
         if show_official:
```

Comment:
does it make sense to only run it on Edu since it may be slow?

Reply:
Ok yes, true. We can do that.